horovod/common/ops/collective_operations.h

```diff
@@ -252,6 +252,20 @@ class AlltoallOp : public HorovodOp {
       return status;
     }
 
+    // Allocate and fill received_splits output
+    if (e.context->framework() == Framework::TENSORFLOW) {
```

Comment:
Have you considered extending support for this to the other supported frameworks?

Reply:
I personally don't have any experience implementing ops in the other frameworks. So this would delay things a bit.

Reply:
No, this isn't blocking, though it is nice to try and keep features across the framework implementations more in sync.

Reply:
OK, I just pushed my go at an implementation for Pytorch.

Giving it some effort, I unfortunately did not manage to install a version of MXNet in my dev environment that would be compatible with the Horovod build. I probably won't find time very soon to focus on this.

Reply:
All right, I managed to sort out my MXNet installation problems and pushed an implementation for that framework, too.